doc/man3/OPENSSL_version_text.pod

```diff
@@ -0,0 +1,186 @@
+=pod
+
+=head1 NAME
+
+OPENSSL_VERSION_MAJOR, OPENSSL_VERSION_MINOR, OPENSSL_VERSION_PATCH,
+OPENSSL_VERSION_PRE_RELEASE, OPENSSL_VERSION_BUILD_METADATA,
+OPENSSL_version_major, OPENSSL_version_minor,
+OPENSSL_version_patch,
+OPENSSL_version_pre_release, OPENSSL_version_build_metadata
+OPENSSL_version_text, OPENSSL_version_text_full, OPENSSL_info
+- get OpenSSL version number
+
+=head1 SYNOPSIS
+
+ #include <openssl/opensslv.h>
+
+ #define OPENSSL_VERSION_MAJOR  x
+ #define OPENSSL_VERSION_MINOR  y
+ #define OPENSSL_VERSION_PATCH  z
+
+ #define OPENSSL_VERSION_PRE_RELEASE "-dev"
+ #undef OPENSSL_VERSION_BUILD_METADATA
+
+ unsigned int OPENSSL_version_major(void);
+ unsigned int OPENSSL_version_minor(void);
+ unsigned int OPENSSL_version_patch(void);
+ const char *OPENSSL_version_pre_release(void);
+ const char *OPENSSL_version_build_metadata(void);
+
+ const char *OPENSSL_version_text(void);
+ const char *OPENSSL_version_text_full(void);
+
+ #include <openssl/crypto.h>
+
+ const char *OpenSSL_info(int t);
+
+=head1 DESCRIPTION
+
+=head2 Macros
+
+The three macros B<OPENSSL_VERSION_MAJOR>, B<OPENSSL_VERSION_MINOR> and
+B<OPENSSL_VERSION_PATCH> represent the three parts of a 3 numbered version
+number, MAJOR.MINOR.PATCH.
+
+The macro B<OPENSSL_VERSION_PRE_RELEASE> is an added bit of text that,
+when defined, indicates that this is a pre-release version (possibly
+development, for which the value is "-dev").
```

Comment:
Will this contain other data like for instance "-alpha2"?

Reply:
That's the idea, yeah